boards/same54-xpro/include/board.h

```diff
@@ -75,6 +75,13 @@ extern "C" {
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name ATA6561 STANDBY pin definition
+ * @{
+ */
+#define AT6561_STBY_PIN            GPIO_PIN(PC, 13)
```

Comment:
sorry i missread the creation of a group for acually documenting the pin (doxygen still thinks it is undocumented)